.gitattributes

```diff
@@ -10,3 +10,6 @@
 *.toml text
 *.txt text
 *.yml text
+
+# Get some basic syntax highlighting
+*.mvir linguist-language=JavaScript
```

Comment:
I think Rust would actually be the best choice here if linguist supports it. Move IR source syntax tries to be consistent with Rust when it makes sense.